native/src/test/java/com/velocitypowered/natives/encryption/VelocityCipherTest.java

```diff
@@ -56,20 +56,18 @@ private void check(VelocityCipherFactory factory, Supplier<ByteBuf> bufSupplier)
     VelocityCipher encrypt = factory.forEncryption(new SecretKeySpec(AES_KEY, "AES"));
 
     ByteBuf source = bufSupplier.get();
-    ByteBuf dest = bufSupplier.get();
-    ByteBuf decryptionBuf = bufSupplier.get();
 
     source.writeBytes(TEST_DATA);
 
+    ByteBuf working_buf = source.copy();
```

Comment:
Checkstyle error. Perhaps `tmpBuf` would be better here.

Reply:
workingBuf is more descriptive.

![](http://theconcordian.com/wp-content/uploads/2018/04/ChangeMyMindONLINE_Zeze_preview.jpeg)